### ISSUE TITLE
I can't extract to normal file

### ISSUE THREAD

**Issue by songsinh88** [ata4/disunity#168]
After use command extract file: java -jar disunity.jar asset unpack sharedassets0.assets. The result like as screen shot attact. Don't understand use command that result normal file. Help me.

<img width="771" alt="screen shot 2016-01-28 at 12 00 34 am" src="https://cloud.githubusercontent.com/assets/4413419/12621236/347bf8a8-c552-11e5-9db7-27695b28cfea.png">

**Comment by a11enwong** [comment]
the same as me

**Comment by ata4** [comment]
Yeah, it's the current behavior in 0.5, you get raw data blocks only. To extract assets, deserialization needs to be re-implemented, which isn't done yet.

**Comment by marcg** [comment]
is there any chance this will be implemented?

**Comment by tsinglee0** [comment]
me too
